packages/react-storybook/.storybook/main.js

```diff
@@ -0,0 +1,11 @@
+const rootMain = require('../../../.storybook/main');
```

Comment:
Will this package ever have stories?

Reply:
maybe in future (to have stories that can be e2e tested)? . it will need a bit overhaul -> migrating from knobs to controls etc

not a big deal - just generator at work ( we can remove it as needed via automation later )

Reply:
This package really doesn't need either storybook config or API Extractor. Adding these files to new packages where they aren't needed is just increasing the maintenance burden later (and in the case of API Extractor configs, unnecessarily increasing build time).

Reply:
> really doesn't need either storybook config

please see previous comment

> and in the case of API Extractor configs, unnecessarily increasing build time

I'd say this statement is a bit exaggerated (can you provide specific data metrics ) ? 

- all new DX packages should be unified as much as possible for a sake of easier automation. API extractor is a must for typescript declaration rolluping. It is to be determined if it's gonna be used also for documentation generation purposes or not.

>  is just increasing the maintenance burden 

quite contrary. with nx and generators everything is automated and kept in sync ✌️

Reply:
> > really doesn't need either storybook config
> 
> please see previous comment

IMO it's a better approach to *not* have extra junk by default, then *only* add it **if/when** needed.

> > and in the case of API Extractor configs, unnecessarily increasing build time
> 
> I'd say this statement is a bit exaggerated (can you provide specific data metrics ) ?

If an API Extractor config is present, the API Extractor generation step runs. Go look for how long that takes in the build. It's probably not huge but cumulatively it is adding a bit of time.

> * all new DX packages should be unified as much as possible for a sake of easier automation. API extractor is a must for typescript declaration rolluping. It is to be determined if it's gonna be used also for documentation generation purposes or not.

Do we actually need declaration rollup for non-component packages?

> > is just increasing the maintenance burden
> 
> quite contrary. with nx and generators everything is automated and kept in sync ✌️

Generators can't magically know which projects should and should not have an API Extractor config.

Reply:
Search build logs for `finished 'api-extractor' in` -- it takes at least a few seconds (more for some packages) because it has to build a TS program for each package, plus however long the analysis takes. The only straightforward way to marginally improve that is not using API Extractor where it's not needed. The more comprehensive but less straightforward way (which we may need to investigate at some point if we keep using the tool and want to invest in improving build perf) is creating a shared TS program object and programmatically invoking any tool that needs to use it, instead of using their CLIs.